lib/sfrest/centralized_role_management.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SFRest
+  # We disable that because we need to keep this naming due to connection.rb way of autoloading stuff and rubocop
```

Comment:
```suggestion
  # We need to keep this naming due to the way  connection.rb autoloads things.
```

Reply:
Can probably leave out the "and rubocop doesn't like it". We can assume that given we're disabling rubocop

Reply:
Fixing :D